testprovider/requirements.txt

```diff
@@ -1,3 +1,3 @@
-django==3.2.23
+django==3.2.24
```

Comment:
We might want to bump this up to Django 4.2 LTS. But I can look at that in a future PR.